src/server/repository/assessmentCycle/links/getMany.ts

```diff
@@ -20,7 +20,7 @@ const _getOrderClause = (
   orderBy: string | undefined,
   orderByDirection: TablePaginatedOrderByDirection | undefined
 ): string => {
-  if (Objects.isEmpty(orderBy)) return ''
+  if (Objects.isEmpty(orderBy)) return 'order by id asc'
```

Comment:
Added a default `order by` to keep the results consistent if no order by is selected.